test/suite/test_compact07.py

```diff
@@ -104,24 +104,24 @@ def test_compact07(self):
         self.session.checkpoint()
 
         # There should be more free space in the last created tables compared to the very first one.
+        min_free_space = 0
         for i in range(self.n_tables):
-            self.assertGreater(self.get_free_space(uri), free_space_20)
+            free_space = self.get_free_space(uri)
+            self.assertGreater(free_space, free_space_20)
+            if i == 0:
+                min_free_space = free_space
+            elif free_space < min_free_space:
+                min_free_space = free_space
```

Comment:
Why is this necessary? The space we're deleting is hard coded.

Reply:
That's true, I was being overly cautious in case the free space is not as much as it should have been.

Reply:
2f72748230f28d1bdf9076116a3b34415bad4d96